README-iso.md

```diff
@@ -0,0 +1,260 @@
+* [ ] multipath support ?
+* [x] getty on tty2 is broken
+* [x] document dependencies (as asserts)
+* [x] check why the installed system is seen as 8.2 (comes from the installer's branding !?)
+
+Work to reproduce install.img
+
+* reproduce XCP-ng 8.2.1 ~+
+  * known issues
+    - [x] check for remaining host contaminations
+    - [x] use debug hypervisor
+  * still some packages missing
+    - [x] host-installer-startup, not provided as RPM in CH, only as SRPM
+    - [x] xen-installer-files
+  * packages replaced in xcp-ng
+    - [x] xsenserver-release* -> xcp-ng-release*
+  * missing files:
+    - [x] /opt/xensource/installer/version.py
+    - [x] installer.service getty@tty2.service
+    - [x] sshd.service.d/installer.conf (disables sshd by default, enables with sshpassword=xxxx)
+    - [ ] ldconfig.service xenstored.service etc. links to /dev/null (those two at least seem
+          unneeded)
+    - [ ] /etc/udev/rules.d/: 11-dm-mpath.rules 62-multipath.rules 69-dm-lvm-metad.rules links to /dev/null
+    - [ ] /etc/udev/rules.d/40-multipath.rules
+    - [ ] /usr/lib/firmware/updates/intel/ice/ddp/ice-1.3.26.0.pkg (we have `ice.pkg`)
+    - [ ] /usr/lib/modules/4.19.227/kernel/drivers/infiniband (removed by host-installer "%triggerin")
+    - [x] /init -> /sbin/init
+    - [x] ~~/bin/support.sh (trivial wrapper around xelogging.py - CH only ?)~~
+  * missing files we don't care to add
+    - /etc/sysconfig/network: no adverse effect noticed during netinstall
+    - getty.target.wants/lvm2* broken links
+    - /usr/sbin/scsi_id -> /lib/udev/scsi_id broken link
+    - /run/reboot-required.d/: kernel/4.19.19-7.0.13 xen/4.13.4-9.17
+  * extra files removed by CH, we don't care to remove (too small to matter)
+    - /usr/lib/dracut
+    - /etc/dracut.conf.d/ (not even removed in 8.3 any more)
+  * missing in installed system
+    - [ ] Citrix rpm keys
+  * still some packages to be cleaned up (50MB)
+    - [x] binutils etc.
+    - [x] python2.7 .pyc and .pyo
+    - [ ] some extra service dependencies
+    - [ ] some /usr/bin binaries from systemd (bootctl, etc.), [dp]gawk, oldfind, info(key), ssh-{agent,keyscan}, ...
+    - [ ] some /usr/sbin binaries
+    - [ ] firwmare: bnx2(x), i915, qed
+    - [ ] drivers: some ethernet (bcom, intel, qlogic, etc), some scsi, some fs, gpu
+    - [ ] udev hwdb files
+    - [ ] /etc/openldap/certs/
+    - [ ] /var/cache
+    - ...
+  * file diffs:
+   * kernel modules differ (!)
+   * /etc/issue
+   * host contamination:
+     * ~~$B/etc/multipath.conf.disabled~~: not really, comes from BuildRequires
+  * metadata diffs
+    * removed locale files are not flagged as missing in CH8C install.img by "rpm -V", not clear now they achieved this
+    * CH8C has changes to filemodes (w and s bits removed), visible with rpm -V.  No clue what they tried to achieve.
+
+
+## from Samuel
+
+Comparison of install.img between XCP-ng 8.2.0's official ISOs and newly generated install.img
```

Comment:
Do we want to keep this title?

Reply:
`grub-mkrescue` is "the reference" in someway, providing largest platform support for booting, but OTOH adds tons of stuff we don't need at all. It was used as a reference to implement UEFI boot, and I kept it to have it handy if/when we need it, since the command options are not that obvious.  Eg. we may want to add support for x86 Macs some day.
Will add a comment.

Reply:
no :)